includes/functions.php

```diff
@@ -0,0 +1,196 @@
+}
+
+/**
+ * Render stories based on customizer settings.
+ *
+ * @return void
+ */
+function render_theme_stories() {
+	//phpcs:disable WordPress.Security.EscapeOutput.OutputNotEscaped
+	echo get_plugin_instance()->customizer->render_stories();
+}
+
+/**
+ * Returns field state for the provided view type.
+ *
+ * @param string $view View Type.
+ *
+ * @return GridView|ListView|CircleView|CarouselView
+ */
+function get_field( $view = 'grid' ) {
+
+	switch ( $view ) {
+		case 'grid':
+			$field_state = new GridView();
+			break;
+		case 'list':
+			$field_state = new ListView();
+			break;
+		case 'circles':
+			$field_state = new CircleView();
+			break;
+		case 'carousel':
+			$field_state = new CarouselView();
+			break;
+		default:
+			$default_field_state = new CircleView();
+			/**
+			 * Filters the fieldstate object.
+			 *
+			 * This depicts
+			 *
+			 * @since 1.3.0
+			 *
+			 * @param FieldState $default_field_state Field states for circle view.
+			 */
+			$field_state = apply_filters( 'web_stories_default_fieldstate', $default_field_state );
+	}
+
+	return $field_state;
+}
+
+/**
+ * Wrapper function for fetching field states
+ * based on the view types.
+ *
+ * Mainly uses FieldState and Fields classes.
+ *
+ * @SuppressWarnings(PHPMD.UnusedLocalVariable)
+ *
+ * @return array
+ */
+function fields_states() {
+	$views = get_layouts();
+
+	$fields = [
+		'title',
+		'author',
+		'date',
+		'image_align',
+		'excerpt',
+		'sharp_corners',
+		'archive_link',
+		'circle_size',
+	];
+
+	$field_states = [];
+
+	foreach ( $views as $view_type => $view_label ) {
+		$field_state = get_field( $view_type );
+		foreach ( $fields as $field ) {
+			$field_states[ $view_type ][ $field ] = [
+				'show'     => $field_state->$field()->show(),
+				'label'    => $field_state->$field()->label(),
+				'readonly' => $field_state->$field()->readonly(),
+			];
+		}
+	}
+
+	return $field_states;
+}
+
+/**
+ * Get supported layouts for web stories.
+ *
+ * @return mixed|void
+ */
+function get_layouts() {
+	/**
+	 * Filter supported layouts.
+	 *
+	 * @since 1.3.0
+	 *
+	 * @param array $layouts Default supported layouts.
+	 */
+	return apply_filters(
+		'web_stories_layouts',
+		[
+			'carousel' => __( 'Box Carousel', 'web-stories' ),
+			'circles'  => __( 'Circle Carousel', 'web-stories' ),
+			'grid'     => __( 'Grid', 'web-stories' ),
+			'list'     => __( 'List', 'web-stories' ),
+		]
+	);
+}
+
+/**
+ * Get supported order by options for web stories.
+ *
+ * @return array
+ */
+function get_stories_order() {
+	/**
+	 * Filter supported order by options.
+	 *
+	 * @param array $orderby Default supported order by options.
+	 *
+	 * @since 1.3.0
+	 */
+	return apply_filters(
+		'web_stories_orderby',
+		[
+			'latest'               => __( 'Latest', 'web-stories' ),
+			'oldest'               => __( 'Oldest', 'web-stories' ),
+			'alphabetical'         => __( 'A -> Z', 'web-stories' ),
+			'reverse-alphabetical' => __( 'Z -> A', 'web-stories' ),
+		]
```

Comment:
I've always found this very confusing, it mixes order and orderby in one list.

We should instead split this up under the hood.

So `alphabetical` becomes `orderby: title, order: ASC`